.github/pull-request-template.md

```diff
@@ -16,3 +18,16 @@ Changes proposed in this pull request:
 **Related issue(s)**
 
 <!-- If you refer to a particular issue, provide its number. For example, `Resolves #123`, `Fixes #43`, or `See also #33`. -->
+
+**Confirm that everything was done**
```

Comment:
what about: `Confirm that your PR meets all requirements`
or, even more simply, why not call it `Definition of done`